Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3176,6 +3176,13 @@ private static void RunInitializeInPlayer()
             #endif
         }
 
+        // Initialization is triggered by accessing InputSystem. Some parts (like InputActions)
+        // do not rely on InputSystem and thus can be accessed without tapping InputSystem.
+        // This method will explicitly make sure we trigger initialization.
+        internal static void EnsureInitialized()
```

Comment:
Not sure I see why invoking this empty method makes a difference?

Is it possible to add some kind of test case to verify this initialization scenario?

Reply:
Calling a method from a class ensures its class constructor is called. `InputSystem` initialization is triggered via this mechanism in general (i.e. call any of its public methods -> initialization automatically happens before actual method invocation).